molgenis/molgenis-emx2
# chore: make sure spotless uses lf

Add .gitattributes so spotless doesn't change all java files on windows to crlf every time it builds. 

how to test:
- run gradlew format on windows
- See that all java files are still lf and don't change to crlf

todo:
